tests_async/system/requests/test_upload.py

```diff
@@ -140,21 +136,23 @@ async def check_content(blob_name, expected_content, transport, headers=None):
 async def check_tombstoned(upload, transport, *args):
     assert upload.finished
     basic_types = (resumable_requests.SimpleUpload, resumable_requests.MultipartUpload)
-    #CHANGE EXCEPTION
+
+    # TODO() CHANGE EXCEPTION
```

Comment:
is this up to date

Reply:
Elaborated on this in the todo in the file. There was an exception raising error earlier, though this may be cleared now with our changes. However, this is somewhat clouded by the timeout context manager bug as the system tests for download/upload currently can only be run in isolation.